src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -49,7 +50,17 @@ export class AddCellAction extends Action {
 	public run(context: INotebookEditor): Promise<boolean> {
 		return new Promise<boolean>((resolve, reject) => {
```

Comment:
While you're here - just make this function async and you can get rid of the wrapped promise and the try/catch

Reply:
Definitely this

Reply:
updated